package.json

```
@@ -24,9 +24,10 @@
     "test:jest": "jest --env=jsdom /test/.*.js?$",
     "test:fix": "npm run test:eslint:fix",
     "test:eslint": "eslint 'src/**/*.{js,vue}'",
-    "test:eslint:fix": "npm run test:eslint -- --fix",
+    "test:eslint:fix": "npm run test:eslint -- --fix --cache",
```

Is there reason to only enable this when in the fixing mode, can we not enable caching for normal checking (line above)?

---

My bad, I add the cache option in the overhead line as well.  Also, it's necessary to enable caching for normal checking as well which helps to improve ESLint's running time by ensuring that only changed files are linted.

---

```suggestion
    "test:eslint:fix": "npm run test:eslint -- --fix",
```
With it added to the line above, it can be removed here, as this line just runs the previous line

---

done